### ISSUE TITLE
barricades are useless

### ISSUE THREAD

**Issue by Gireen** [Unvanquished/gameplay#6]
humans can just jump over them with wall jumps

**Comment by illwieckz** [comment]
what if barricades trap touching humans 👀

it's not like if they didn't have many arms…

**Comment by DolceTriade** [comment]
barricade + trapper combo building seems like a fun idea

**Comment by Viech** [comment]
Claiming that barricades are useless because there is one particular gameplay element that negates one of the things they are doing doesn't qualify as an issue worth tracking in my opinion.

@Gireen you could have said "I personally do not build barricades because I do not feel their effect is worth their cost since when I'm a human I have a really easy time negating their effect.", or something like that. I feel we should require input in that form and close issues like this one on sight.

**Comment by Gireen** [comment]
@Viech  i would say blocking the way for humans is the main purpose of barricades 😃  and i witnessed it on both sides that they don't pose much hindrance for humans with being able to just jump over them as the biggest problem. Cost value discussions are maybe something for another issue.

The ability the trap close humans would in every case solve this problem and be also something interesting. 
Jetpacks and Battlesuits could also be immune to this effect.

**Comment by slipher** [comment]
Barricades trapping or slowing humans that step onto them sounds nice, although it's hard to imagine how exactly it should work.

**Comment by Viech** [comment]
I've hidden all comments; three were discussing solution propsals and one was my rant about the title and a response.

Let's use this issue only to track whether we believe that the introduction of wall jumping warrants a buff to the barricade.

**Comment by Viech** [comment]
If we keep walljumping, then I'd fully agree that the barricade needs a buff. Circumventing a barricade should take movement that isn't entirely standard. Are we happy with the introduction of wall jumping, though? Maybe this is the real culprit. I haven't played enough recently to have a strong opinion on this.

**Comment by Viech** [comment]
@illwieckz, @DolceTriade, @slipher, may I read your proposals as you agreeing that either the barricade or the wall jumping needs to change? (And which one would you prefer?)

**Comment by slipher** [comment]
> @illwieckz, @DolceTriade, @slipher, may I read your proposals as you agreeing that either the barricade or the wall jumping needs to change? (And which one would you prefer?)

Yes, I'd agree with that. Another possible for getting over the barricade which already existed is to jump on it when there is a passing alien, but timing this seems difficult enough that it is not too easy for humans.

Walljumping is fun, but if we decided it had to go, I would be OK with that.

**Comment by Viech** [comment]
I always liked how you could trade some health to get over a cade by jumping on top of an acid tube. Having ways to bypass it is fine as long as they are situational or costly. Maybe all the barricade needs is to be harder to kill so that humans are forced into movement options (if available) for as long as they ignore it.

**Comment by DolceTriade** [comment]
I think a more interesting barricade could be cool :shrug:

**Comment by bmorel** [comment]
BTW I'm ok for removal of the wallrun feature of humans. Or to prevent it from being used on alien buildables which are in practice (lore-wise) living beings able to move, and not concrete or steel hard walls that you can do climbing tricks on them (wallrunning is an IRL climbing technics, after all).

**Comment by illwieckz** [comment]
Same, I'm OK to disable wallruning on alien buildables (and it works on creep, disable it on creep as well).

**Comment by bmorel** [comment]
Disabling it on alien buildables is technically feasible, disabling on creep... I have no idea how to do that. I'm not even sure how creep works exactly. To me, it's likely to be a distance from the creep generator, and not tracing, since creep effect works through walls.
No idea how the creep's plane is checked.

In any cases, wallrun on creep is less a problem imo.

**Comment by bmorel** [comment]
It has to be noted that no mechanics change is actually needed to resolve this, as we already have something that can fix that: creep-slow.
From my experiments, it seems a wide enough creep radius makes it much, much harder to successfully achieve a wallrun over a barricade.

**Comment by Viech** [comment]
I haven't heard anyone defend the ability to walljump over a barricade and there are all sorts of ideas in here on how to limit walljumping or improve the barricade, so I conclude that the problem **as stated in the title** is agreed upon. The remainder should be discussed in idea issues, as there are at least two distinct routes to go.
